packages/blocks/src/root-block/widgets/linked-doc/import-doc/import-doc.ts

```diff
@@ -242,14 +257,15 @@ export class ImportDoc extends WithDisposable(LitElement) {
     const pageIds: string[] = [];
     for (const file of files) {
       const text = await file.text();
+      const [fileName] = file.name.split('.');
```

Comment:
Maybe `file.name.split('.').slice(0, -1).join('.')` would be more accurate?

Reply:
thanks for your replay!